extra/pom.xml

```diff
@@ -120,6 +121,11 @@
                 <artifactId>commons-compress</artifactId>
                 <version>${commons.compress.version}</version>
             </dependency>
+            <dependency>
+                 <groupId>commons-logging</groupId>
+                 <artifactId>commons-logging</artifactId>
```

Comment:
Leaving a separate remark. This looks unnecessary. This dependency can be excluded from other dependencies that rely on it without any visible issues.